LICENSE

```diff
@@ -176,7 +176,7 @@
    END OF TERMS AND CONDITIONS
 
 
-   Copyright 2016 CloudByte, Inc.
+   Copyright 2016-present CloudByte, Inc.
```

Comment:
present is not required .. Do something like this ```Copyright 2016 MayaData, Inc```
cc @kmova  ?

Reply:
Sure! Gimme a few minutes